modules/local/create_gene_heatmap/meta.yml

```diff
@@ -0,0 +1,43 @@
+name: create_gene_heatmap
+description: Generate a gene count heatmap based on annotated and filtered gene counts data.
+keywords:
+  - bioinformatics tools
+  - data visualization
+  - gene heatmap
+tools:
+  - create_gene_heatmap:
+      description: |
+        The create_gene_heatmap module generates a heatmap of gene counts.
+        It supports gene selection via additional files and provides a log-transformed visualization of the data.
+input:
+  - - annotated_counts:
```

Comment:
```suggestion
  - annotated_counts:
```

Reply:
This one I think is actually correct, from looking at other implementations of the meta.yml file (i.e [MultiQC ](https://github.com/nf-core/modules/blob/master/modules/nf-core/multiqc/meta.yml) or [bwa index](https://github.com/nf-core/modules/blob/master/modules/nf-core/bwa/index/meta.yml) ). The way it is typically done, we use double hyphen for each of the input channels and a single hyphen for each subsequent item of the same channel.